logs/options.go

```diff
@@ -0,0 +1,154 @@
+package logs
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/olebedev/when"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	defaultCount      = 100
+	defaultConfigFile = "~/.swo-cli.yaml"
```

Comment:
Should this be .yml?

```suggestion
	defaultConfigFile = "~/.swo-cli.yml"
```